Maple2.Model/Metadata/SetItemOptionMetadata.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Maple2.Model.Enum;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Maple2.Model.Metadata;
+
+public record SetItemInfoMetadata(
+    int Id,
+    int[] ItemIds,
+    int OptionId,
+    string Name);
+
+public record SetItemOptionMetadata(
+    int Id,
+    SetBonusMetadata[] Parts);
+
+public record SetBonusMetadata(
+    int Count,
+    int[] AdditionalEffectIds,
```

Comment:
combine these into a struct since they are supposed to be paired. example: https://github.com/kOchirasu/Maple2/blob/12b1718adf0fc954ea35c1491a0879430cbba5da/Maple2.File.Ingest/Mapper/MapMapper.cs#L85-L86


```cs
public record SetBonusAdditionalEffect(int Id, short Level);
```